internal/services/springcloud/spring_cloud_service_resource_test.go

```diff
@@ -356,7 +389,11 @@ resource "azurerm_spring_cloud_service" "test" {
 func (SpringCloudServiceResource) virtualNetwork(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
This is a bug for how we delete resource groups. For whatever reason, virtual networks are kicking around for awhile. We should remove this since it's unrelated to your changes and we can use the failing test to track down what is happening when we delete a resource group